cmd/oras/root/pull.go

```diff
@@ -156,13 +156,9 @@ func (po *pullOptions) doPull(ctx context.Context, src oras.ReadOnlyTarget, dst
 	}
 
 	var tracked track.GraphTarget
-	if po.TTY != nil {
-		tracked, err = track.NewTarget(dst, "Downloading", "Downloaded ", po.TTY)
-		if err != nil {
-			return ocispec.Descriptor{}, false, err
-		}
-		defer tracked.Close()
```

Comment:
The equivalence of `defer tracked.Close()` is not reflected in the modified code. This is also not observed in `attach` and `push`.

Reply:
`doPush` will handle the closing for `attach` and `push`